composer.json

```diff
@@ -10,12 +10,12 @@
     }
   ],
   "require": {
-    "consolidation/robo": "^4.0"
+    "consolidation/robo": "^4.0 || ^5.0"
   },
   "require-dev": {
     "drupal/coder": "^8.3.16",
     "phpunit/phpunit": "^9.5.5",
-    "phpro/grumphp": "^1.14"
+    "phpro/grumphp": "^1.14 || ^2.6"
```

Comment:
what bring 2.6 that we need and 2.5 or lower does not?

Reply:
True, minimum is 2.3 that brings support for Symfony 7 and PHP 8.3, fixed.